Gopkg.toml

```diff
@@ -53,3 +53,7 @@
 [[constraint]]
   name = "github.com/alexellis/go-execute"
   version = "0.3.0"
+
+[[constraint]]
+  name = "github.com/google/go-querystring"
```

Comment:
Where are we using querystring? I couldn't see it in the diff

Note that url.Parse also offers querystring parsing and is in stdlib

Reply:
we are using it inside `addQueryParams` functions.

Reply:
Please can you use `url.Parse` instead and set the value on the query property? https://golang.org/pkg/net/url/#URL.Query

Reply:
(Instead of this dependency)

Reply:
Yes that can be done as well, but just wanted to check if there is any specific reason we don't want to add a dependency? This is fairly very small one.

Reply:
@viveksyngh it seems like the primary use for this library is that it allows passing the `NamespaceParam` struct as an arg. But wouldn't it be easier to pass a `map[string]string`?  Maybe explaining this struct and how you seeing it evolve could help clarify if we want to user this new library https://github.com/openfaas/faas-cli/pull/753/files#diff-aa6ab6c842e3599a6d4b87d4ac89f6dbR17

Reply:
Idea is to allow any struct can be used to construct the query parameters which has `url` annotation. Also, if we have some query parameters common for few APIs we can create a struct and use that for creating the query parameters rather than creating a map with key-value pairs at multiple places. Managing the `key` for query param become difficult in my opinion and also looked like a cleaner approach to me for query parameters. 

This can be done with maps as well. I am happy to switch to maps and remove this dependency if above statements does not justify the usage.

Reply:
@alexellis what do you think?  I do like that these parameters can be documented on the struct

Reply:
My message might have been lost in translation or too indirect, but I've said a couple of times that I'd like to see net.URL used to handle querystring here. Why? We use everywhere else and it is better for consistency and for maintainability. I might be missing something.

Reply:
I have removed this dependency and using map and `net.URL` package.